bentoml/_internal/runner/container.py

```diff
@@ -96,19 +96,19 @@ def batches_to_batch(
         indices = list(
             itertools.accumulate(subbatch.shape[batch_dim] for subbatch in batches)
         )
-        indices.pop()
+        indices = [0] + indices
```

Comment:
I made this changes so every container type has consistent indices format returned

Reply:
don't we still need the `indices.pop` here? cc @bojiang @sauyon

Reply:
don't we still need the `indices.pop` here? cc @bojiang @sauyon